modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/behavior/TableViewBehaviorBase.java

```diff
@@ -146,20 +148,21 @@ public TableViewBehaviorBase(C control) {
                 new KeyMapping(PAGE_UP, e -> scrollUp()),
                 new KeyMapping(PAGE_DOWN, e -> scrollDown()),
 
-                new KeyMapping(LEFT, e -> selectLeftCell()),
-                new KeyMapping(KP_LEFT, e -> selectLeftCell()),
-                new KeyMapping(RIGHT, e -> selectRightCell()),
-                new KeyMapping(KP_RIGHT, e -> selectRightCell()),
+
+                new KeyMapping(LEFT, (rtl? e -> selectRightCell() : e -> selectLeftCell())),
```

Comment:
... so a keyMapping based on the initial condition will fail after a change of orientation, or what am I missing?

(aside: I really hate this review ui, can't seem to make it take several comments ... argghh)

Reply:
To add multiple comments, go to the `Files changed` tab and click on a line of code to add a comment there. Select `Start a review` rather than `Add single comment`. This way you can batch up multiple comments. Then you select `Finish Review` in the upper right.

Reply:
thanks, Kevin - that's what I tried, but it refused to accept the second .. so sticking with single comments until I feel like digging into this weirdness.

Reply:
@kevinrushforth, this new KeyMapping suggestion does not compile.
We need something like - 
`new KeyMapping(LEFT, e -> { if(isRTL()) selectRightCell(); else selectLeftCell();}),`